pkg/utils/slice.go

```diff
@@ -6,19 +6,6 @@ import (
 )
 
 func DedupeSlice[T cmp.Ordered](s []T) []T {
-	if len(s) < 2 {
-		return s
```

Comment:
returning directly on len(s) < 2 is worth keeping

Reply:
`Compact` does already

Reply:
True and sort is pretty much nop if len = 1, so can get rid of it.

Reply:
Right, yeah.